include/graphblas/reference/vector.hpp

```diff
@@ -855,6 +855,24 @@ namespace grb {
 #endif
 			}
 
+			/**
+			 * Creates a reference vector with given values.
+			 * This constructor takes an initializer list of values that will be copied.
```

Comment:
As above, I don't think the generic comments are required here and could be removed

As an internal comment though,maybe good to add: even though this constructor is likely called by other backends (i.e., by BSP1D, or by nonblocking), it is not useful to have a variant where the initaliser list is passed by as a const-reference, because the C++ specification of an initialiser list makes sure it's a lightweight object (e.g., a pointer).